kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/service/SpecManagementServiceImpl.java

```diff
@@ -128,6 +128,16 @@ public synchronized void saveServerTemplate( ServerTemplate serverTemplate ) {
         }
 
         notificationService.notify( new ServerTemplateUpdated( serverTemplate ) );
+
+        Collection<ContainerSpec> containerSpecs = serverTemplate.getContainersSpec();
+        if (containerSpecs != null && !containerSpecs.isEmpty()) {
+            for (ContainerSpec containerSpec : containerSpecs) {
+                if (containerSpec.getStatus().equals(KieContainerStatus.STARTED)) {
+                    List<Container> containers = kieServerInstanceManager.startContainer( serverTemplate, containerSpec );
+                    notificationService.notify( serverTemplate, containerSpec, containers );
```

Comment:
I assume the "notification service" also checks the ```ContainerSpec``` and starts the servers if applicable?